### DISCUSSION TITLE
CMS

### DISCUSSION THREAD

**Discussion by francescogatto** [ImaginativeShohag/Why-Not-Compose#43] [General]
What is the CMS package? how to run him? Thanks a lot

**Comment by ImaginativeShohag** [comment]
The app includes the CMS module: https://github.com/ImaginativeShohag/Why-Not-Compose/tree/main/cms

Open the app in Android Studio and follow the [setup](https://github.com/ImaginativeShohag/Why-Not-Compose#setup).

Here is a quick setup flow:

In your `local.properties` file add following keys:

```
MAPS_API_KEY=MAP_KEY
CMS_API_KEY=CMS_KEY
```

`MAP_KEY` can be blank no issue.

Replace `CMS_KEY` with your [GoRest](https://gorest.co.in/) API key. It is free, just register and you will get API key.
